src/components/sectionList/SectionListWrapper.tsx

```diff
@@ -24,19 +29,29 @@ type SectionListWrapperProps = {
     data: ModelCollection<BaseListModel> | undefined
     pager: Pager | undefined
     error: FetchError | undefined
+    refetch: () => void
 }
 
 export const SectionListWrapper = ({
     data,
     error,
     pager,
+    refetch,
 }: SectionListWrapperProps) => {
     const { columns: headerColumns } = useModelListView()
     const schema = useSchemaFromHandle()
     const { selectedModels, checkAllSelected, add, remove, toggle, clearAll } =
         useSelectedModels()
     const [detailsId, setDetailsId] = useState<string | undefined>()
     const [sharingDialogId, setSharingDialogId] = useState<string | undefined>()
+    const deleteModelMutation = useDeleteModelMutation(schema)
+    const deleteModel = useCallback(
+        async ({ id, displayName }: BaseListModel) => {
+            await deleteModelMutation.mutateAsync({ id, displayName })
```

Comment:
Could call mutate with onError instead of async to avoid needing to think about catching error here as well.